File: Crossplatform/c_sharp_version/Middleware/domain/LoginLogger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    [System.Serializable]
    public class LoginLogger
    {
        public string onlineUser;
      public LoginLogger() 
     { 
            this.onlineUser = null;
     }
     public LoginLogger(string onlineUser)
        {
            this.onlineUser=onlineUser;
        }
    }
}

File: Hibernatfe Java HW/Middleware/src/main/java/domain/LoginLogger.java
package domain;

import java.io.Serializable;

public class LoginLogger implements Serializable {
    public String onlineUser;
    public LoginLogger(String onlineUser){
        this.onlineUser = onlineUser;
    }
    public LoginLogger(){
        this.onlineUser = null;
    }
}
